relay/relay.go

```diff
@@ -496,7 +496,7 @@ func (r *Relay) handleEvents(ctx context.Context, reader2 reader.Reader, transfo
 		}
 
 		if needSavePos {
-			err = r.SaveMeta(lastPos, lastGTID)
+			err = r.SaveMeta(lastPos, lastGTID.Clone())
```

Comment:
Possible performance regression. But I have no idea.

Reply:
How about we merging this PR now? I met this DATA WARN again